Rakefile

```diff
@@ -35,9 +35,15 @@ task 'lint:cucumber' do
 end
 
 
+# Unit tests
+desc 'Run unit tests'
+task 'unit-test' do
+  sh 'bin/unit_test_go'
```

Comment:
Here is how I name tests in other projects:
- unit tests: `bin/tests`
- feature tests: `bin/features`
- all tests: `bin/spec`

Advantage is less typing in the terminal. If you have a different opinion, let's talk.  :)